index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Vite + React + TS</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@mdi/font/css/materialdesignicons.css"/>
```

Comment:
Can we re-use our dependency on FontAwesome instead of bringing in a new icon set?

Reply:
sort icons don't exist in Fontawesome icon in screenshots

Reply:
Sounds good, thanks for the info!